app/services/notices/setup/abstraction-alerts/submit-alert-email-address.service.js

```diff
@@ -49,10 +57,12 @@ function _validate(payload) {
     return null
   }
 
-  const { message } = validation.error.details[0]
+  const { message, path } = validation.error.details[0]
 
   return {
-    text: message
+    text: message,
+    radioFormElement: path[0] === 'alertEmailAddress' ? { text: message } : null,
```

Comment:
path[0] ? 

Needs a better name or explanation. 

Should be another way to do it if they are both the same for different thing.